portal-impl/src/portal.properties

```diff
@@ -6747,7 +6747,7 @@
     #
     # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD_UI_PERIOD_VISIBLE_OPENBRACKET_DEV_CLOSEBRACKET_
     #
-    feature.flag.ui.visible[dev]=false
+    feature.flag.ui.visible[dev]=true
```

Comment:
This cannot default to true. That should be done inside of `portal-ext.properties`.

Reply:
Sorry, just something I was testing. This commit is a mess, just a draft